presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/Environment.java

```diff
@@ -322,6 +322,15 @@ public String getEnvironmentName()
             return name;
         }
 
+        public Builder containerDependsOn(String container, String otherContainer)
```

Comment:
parameter naming does not make it obvious which container depends on which.
Maybe name `container` and `dependencyContainer`?